composefiles/swarm-uniflow-micros.yml

```diff
@@ -11,7 +11,7 @@ services:
     environment:
       - UNICONFIG_URL_BASE=https://${CONSTRAINT_HOSTNAME}_uniconfig:8181/rests
     healthcheck:
-      test: cat /home/app/healthcheck
+      test: curl --insecure --silent --write-out 'HTTPSTATUS:%{http_code}' -X GET http://conductor-server:8080/api/health
```

Comment:
we should also keep old healthcheck there. So service is healthy when both calls will be successful